src/service/dashboards/timed_annotations.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright 2024 OpenObserve Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use config::meta::timed_annotations::{
+    TimedAnnotation, TimedAnnotationDelete, TimedAnnotationReq, TimedAnnotationUpdate,
+};
+use infra::table;
+
+#[tracing::instrument]
+pub async fn create_timed_annotations(
+    dashboard_id: &str,
+    req: TimedAnnotationReq,
+) -> Result<Vec<TimedAnnotation>, anyhow::Error> {
+    let timed_annotations_res =
+        table::timed_annotations::add_many(dashboard_id, req.timed_annotations, false).await?;
+
+    #[cfg(feature = "enterprise")]
+    for timed_annotation in timed_annotations_res.iter() {
+        match super_cluster::emit_timed_annotation_create_event(
+            dashboard_id,
+            timed_annotation.clone(),
+            true,
+        )
```

Comment:
Since we're always setting the `use_given_id` parameter to `true` when we emit a `DashboardMessage::TimedAnnotationCreate`, I think we could just remove that field from the `DashboardMessage::TimedAnnotationCreate` enum variant entirely.

Reply:
yeah, that makes sense.